src/components/common/ui/link/Link.tsx

```diff
@@ -15,8 +15,12 @@ const CustomLink: React.FC<CustomLinkProps> = ({
   ...rest
 }) => {
   return (
-    <Link legacyBehavior {...rest}>
-      <a className={mergeClassNames(styles['link'], className)}>{text}</a>
+    <Link
+      legacyBehavior
```

Comment:
1. Delete `legacyBehavior`
2. You didn't rewrite `<a>` in `Breadcrumbs`
3. Don't forgot to check elements that you change, because this link is broken on the site
![image](https://github.com/fictadvisor/fictadvisor-web/assets/102873980/2a72243f-7dd0-472d-ac64-a33cc148e05f)